decidim-admin/spec/commands/decidim/admin/destroy_component_spec.rb

```diff
@@ -38,8 +38,8 @@ module Decidim::Admin
         subject.call
 
         component = results[:component]
```

Comment:
I think the intention of this spec would be to test against this:
https://github.com/decidim/decidim/blob/5133ff34701f22dcf4e390a9aa01a7b4b1dc982c/decidim-admin/spec/commands/decidim/admin/destroy_component_spec.rb#L9

The changes proposed here are not changing anything in the spec, the `RSpec/IdenticalEqualityAssertion` still would apply here although this change would make the underlying problem hidden.